metron-platform/metron-parsers/src/test/java/org/apache/metron/parsers/integration/ParserDriver.java

```diff
@@ -0,0 +1,168 @@
+/**
```

Comment:
As I see it, there are a couple of things we want to test for every parser:
* The parser works (i.e. the unit tests we currently have)
* The parser can be driven from within the `ParserBolt` (the current integration tests as of this PR)

Beyond that, I think it's reasonable to have a `ParserIntegrationTest` which just validates data flows through the parser topology assuming a correctly working parser and just validates that things work within Storm.  I would not expect every parser to require this kind of test.  We have something similar to that in `SimpleHbaseEnrichmentWriterIntegrationTest`.